lua/lspconfig/configs.lua

```diff
@@ -37,13 +37,13 @@ function configs.__newindex(t, config_name, config_def)
       filetypes = { config.filetype, 't', true },
       on_new_config = { config.on_new_config, 'f', true },
       on_attach = { config.on_attach, 'f', true },
-      commands = { config.commands, 't', true },
+      user_commands = { config.user_commands, 't', true },
     }
-    if config.commands then
-      for k, v in pairs(config.commands) do
+    if config.user_commands then
+      for _, v in pairs(config.user_commands) do
         validate {
-          ['command.name'] = { k, 's' },
-          ['command.fn'] = { v[1], 'f' },
+          ['command.name'] = { v.name, 's' },
+          ['command.fn'] = { v.command, 'f' },
```

Comment:
For `config_def.user_commands` this is called `comand.command` ... maybe using a term avoids confusion.

Reply:
I feel like I didn't push a commit as a I did a search/replace renaming for these, I'll fix it